client/landing/stepper/declarative-flow/flows/onboarding/onboarding.ts

```diff
@@ -60,43 +51,23 @@ const onboarding: Flow = {
 	isSignupFlow: true,
 	__experimentalUseBuiltinAuth: true,
 	useTracksEventProps() {
-		const [ isLoading, isGoalsAtFrontExperiment ] = useGoalsFirstExperiment();
 		const userIsLoggedIn = useSelector( isUserLoggedIn );
-		const goals = useSelect(
-			( select ) => ( select( ONBOARD_STORE ) as OnboardSelect ).getGoals(),
-			[]
-		);
 
 		// we are only interested in the initial values and not when they values change
-		const initialGoals = useRef( goals );
 		const initialLoggedOut = useRef( ! userIsLoggedIn );
 
 		return useMemo(
 			() => ( {
-				isLoading,
 				eventsProperties: {
 					[ STEPPER_TRACKS_EVENT_SIGNUP_START ]: {
-						is_goals_first: isGoalsAtFrontExperiment.toString(),
-						...( isGoalsAtFrontExperiment && { step: 'goals' } ),
 						is_logged_out: initialLoggedOut.current.toString(),
```

Comment:
I would probably remove this too. It was added [here](https://github.com/Automattic/wp-calypso/pull/98097). This would mean getting rid of the whole hook.